### PR TITLE
Rebasing some personal clean up work + adding eslintrc :wrench:

### PR DESCRIPTION
**Pull request by bnookala**
- Clean up code found to be smelly by eslint

@brendandburns, @itowlson: I cherry-picked one (of many) commits onto master, and merged from there. My next pull will attempt to modularize the code.

### CODE REVIEW

File: explainer.js [hunk 1]
```diff
@@ -64,11 +65,11 @@ function formatDescription(lines) {
         field1 <type>
         first line
         second line
-    
+
```
Comment (itowlson) [inline explainer.js]: These comments are describing the output of `kubectl explain`. If you add blank lines to them, they'll no longer be accurate (e.g. a maintainer might think they can rely on a blank line in the input).  I'd very much prefer to keep the comments as they are.

Comment (bnookala) [inline explainer.js]: @itowlson I think this change was actually just removing extraneous spaces and a new line and replacing them with just a single new line - is that acceptable? You can confirm that I didn't add any newlines:

Master: https://raw.githubusercontent.com/brendandburns/vs-kubernetes/master/explainer.js
My branch: https://raw.githubusercontent.com/bnookala/vs-kubernetes/938ba8616074c7fb405ba01a8b0fffd70b715f7e/explainer.js

Comment (itowlson) [inline explainer.js]: Mea culpa, you are absolutely right.  I misread the way GitHub presented the diff.  Apologies!

**Comment by brendandburns** [comment]
@bnookala thanks for the PR, I merged @itowlson one first since it was earlier in the queue, I think you need to rebase now.

Apologies for the churn!

**Comment by bnookala** [comment]
@brendanburns no worries! just did a rebase - let me know if you've any questions about this.

**Comment by brendandburns** [comment]
Sorry, needs a rebase again. I promise this is the next PR I will merge...

**Comment by bnookala** [comment]
@brendandburns just did another rebase.

**Comment by itowlson** [comment]
@brendandburns I'll hold off rebasing my other PR - keen to see @bnookala's changes hit master (and the Typescript conversion would be great).

**Comment by brendandburns** [comment]
merged. Thanks! would love to do the typescript conversion next...

We should also add some automation so that we eslint the PR code before merge so we stay eslint clean.

**Pull request merged**